src/main/steps/tasklistresponse/proceedings/proceedingUtils.ts

```diff
@@ -80,8 +80,19 @@ export const OPotherProceedingsSessionParserUtil = (UserCase, keys, URLS, sessio
   if (UserCase.hasOwnProperty(sessionKey)) {
     const orderSessionStorage = [] as { key: string; valueHtml: string; changeUrl: string }[];
     UserCase[sessionKey].forEach(order => {
-      if (UserCase['otherProceedings']?.['order'].hasOwnProperty(`${order}s`)) {
-        const orderDetails = UserCase['otherProceedings']?.['order'][`${order}s`];
+      if (
```

Comment:
Why is this condition put?

Reply:
This is also same as the above one... I just destructured what ever variable necessary for proceedings and using them.